src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            Ball ball = new Lottery().getRandomBall();
+            balls[i] = ball;
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. You can create a single instance of `Lottery` before the loop and use it to generate all `Ball` objects.